sdk/dotnet/Compute/GetResourcePolicy.cs

```diff
@@ -66,42 +66,43 @@ public GetResourcePolicyArgs()
     [OutputType]
     public sealed class GetResourcePolicyResult
     {
-        /// <summary>
-        /// Description of this Resource Policy.
-        /// </summary>
-        public readonly string Description;
```

Comment:
`Description` property is gone?

Reply:
This is actually a property that never existed upstream in the SDK and thus was never actually set so it's more of a cleanup on the TF side